app/controllers/subject_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
```

Comment:
filename should be `subjects_controller` (plural form)

Reply:
Yes, you are right, just fixed that, it's a mistake when I was generating controller